src/api/app/views/webui/attribute/_issue_fields.html.haml

```diff
@@ -0,0 +1,6 @@
+.nested-fields
+  .field
+    = f.select :issue_tracker_id, IssueTracker.all.collect { |tracker| [ tracker.name, tracker.id ] }.sort, include_blank: false
```

Comment:
But if you see some things that can be refactored why not just refactor it now? Otherwise what happens is we say "the point of this task is not to refactor just to fix/do X... we will refactor it later" then the code never gets refactored and we end up with more and more technical debt..